tests/integration/questionnaire/test_questionnaire_list_collector.py

```diff
@@ -206,8 +204,19 @@ def test_list_summary_on_question(self):
         self.post({"another-anyone-else": "No"})
 
         self.assertInBody("Are any of these people related to one another?")
+
         self.assertInBody("Marie Claire Doe")
 
+        self.post({"radio-mandatory-answer": "No, all household members are unrelated"})
+
+        self.assertInUrl("/sections/section/")
+
+        self.post()
+
+        self.assertInUrl("/summary/")
+
+        self.assertNotInBody("£12.00")
```

Comment:
This is not testing "list summary on question" and should be moved into it's own test.